### ISSUE TITLE
Teleport Near Oculus/chest instead of exact location

### ISSUE THREAD

**Issue by thelostabbey** [Akebi-Group/Akebi-GC#517]
I was wondering if we could do a teleport near chest/oculus (especially oculus).  Sometimes when it teleports me to an oculus or chest I'll end up under water in a locked area and for oculus you will auto-loot on teleport.  It might look kinda sketch to loot an item that is locked behind a quest. if we can get a teleport near item outside of loot range that would be amazing.

**Comment by Taiga74164** [comment]
then just manually teleport there with Map Teleport

**Comment by thelostabbey** [comment]
@Taiga74164 fair enough.  I do use that feature regularly but it's hard to pinpoint within a safe distance especially if you are trying to track down missing Oculi or if the Oculi is obscured underground or in a puzzle.  I was thinking something that complimented the ESP+hotkey.  Anyway, if you do or don't, thanks for your time!  Everything is awesome.

**Comment by Taiga74164** [comment]
well you tell me how can we determine what you think is "safe" distance since not all positions are the same. you might as well just create a TP location modified for yourself.

**Comment by thelostabbey** [comment]
> well you tell me how can we determine what you think is "safe" distance since not all positions are the same. you might as well just create a TP location modified for yourself.

I was just hoping to add a user input static value to the x-coordinates and wasn't expecting devs to find any complicated solution.  if it teleports me into a mountain then so be it.  Unfortunately I don't know how the oculus teleport calculates the vectors.  If you can just point me where the vectors are calculated I can look at doing a dirty custom one like in "map teleport" example below for myself.  Again, thank you for all your help.  

![tele2](https://user-images.githubusercontent.com/48223004/186836544-006e215f-3129-418f-8a6c-03f95ab8af99.jpg)

**Comment by llsf88** [comment]
my way of doing this is enable ESP and teleport with map near it, then with ESP I can see if it is somewhere I should not go
